mongo/job_store.go

```diff
@@ -167,7 +168,18 @@ func (m *JobStore) GetJobs(ctx context.Context) (models.Jobs, error) {
 		return results, err
 	}
 
+	paginator := pagination.NewPaginator(m.cfg.DefaultLimit, m.cfg.DefaultOffset, m.cfg.DefaultMaxLimit)
+	offset, limit, err := paginator.ValidatePaginationParameters(offsetParam, limitParam, numJobs)
+	if err != nil {
+		return results, err
```

Comment:
should we capture error details here using errorf?

Reply:
My understanding is that we should either log errors or return them but there's no need to do both. The details of these errors get captured in the handler and dealt with by sending the appropriate error code and message back in the response.